src/Random.js

```diff
@@ -235,6 +235,11 @@ function Random({ isAuthenticated, org, orgSlug, person }) {
     }
 
     if (hint.type === HINT_SITE) {
+      if (
+        person.blog.indexOf("http://") != 0 &&
+        person.blog.indexOf("https://") != 0
+      )
```

Comment:
Sorry for nitpicking, but could you please change `!=` to a strict comparison (`!==`) and add curly braces around the expression, for consistency with the rest of the codebase?

Reply:
hmm, you mean around the statement?